backend/src/main/kotlin/com/group7/artshare/DTO/ArtItemDTO.kt

```diff
@@ -19,7 +18,7 @@ class ArtItemDTO : Comparable<ArtItemDTO> {
     var ownerAccountInfo : AccountInfo? = null
     var ownerId : Long? = null
     var onAuction : Boolean? = null
-    var auction : Auction? = null
+    var maxBid : Bid? = null
```

Comment:
This should be BidDTO

Reply:
updated with the latest commit, thanks for pointing out.